data/sql/updates/pending_db_world/rev_1616252519696041200.sql

```diff
@@ -0,0 +1,257 @@
+(18226, 0, 0, 0, 4, 0, 100, 1, 0, 0, 0, 0, 11, 32020, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Talbuk Patriarch - On Aggro - Cast \'32020\' (No Repeat)'),
+(18226, 0, 1, 0, 0, 0, 100, 0, 2500, 5000, 20000, 25000, 11, 32019, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Talbuk Patriarch - In Combat - Cast \'32019\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17158;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17158);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17158, 0, 0, 0, 2, 0, 100, 0, 20, 80, 0, 0, 11, 32017, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Dust Howler - Between 20-80% Health - Cast \'32017\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 18205;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 18205);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(18205, 0, 0, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 32023, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Clefthoof - Between 20-80% Health - Cast \'32023\' (No Repeat)');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17130;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17130);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17130, 0, 0, 0, 0, 0, 100, 0, 2500, 5000, 20000, 25000, 11, 32019, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Talbuk Stag - In Combat - Cast \'32019\'');
```

Comment:
It already has AI

Reply:
I don't know if it's better than it is now

Reply:
Changed cause Gore duration was overlaping.